pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/FormComponent.java

```diff
@@ -44,6 +48,14 @@ public boolean shouldHideNumber() {
         return hideDisplayNumber;
     }
 
+    @Override
+    public void registerTemplateIds(Consumer<Long> registry) {
+    }
+
+    @Override
+    public void applyRenderedTemplates(Provider<String> rendered, ContentStyle style) {
+    }
```

Comment:
Looks like these methods have nothing to do in `FormComponent`. Should we just leave empty method bodies here for the future or it makes sense to implement `Renderable` in `PhysicianInstitutionComponent` directly?

Reply:
I think we should declare `Renderable` here since all `FormComponent`s should hook into this functionality. Since `FormComponent` is an abstract class, maybe we just declare it and not add overrides here? So that child classes should implement them?